### ISSUE TITLE
convert_weights 

### ISSUE THREAD

**Issue by joefang89** [chuckcho/c3d-tensorflow2#2]
@chuckcho 
Hi Chuck,

Thanks a lot for sharing the tensorflow version of C3D. I met a problem when try to run "convert_weights.py". The weight and bias of conv layer are 4 dim instead of 5 dim as shown below. But in the following transpose action, weight and bias are treated as 5 dim. Could you please check it? 

Thanks a lot.

conv1a: w_shape, b_shape=(64, 3, 3, 3), (1, 1, 1, 64)
conv2a: w_shape, b_shape=(128, 64, 3, 3), (1, 1, 1, 128)
conv3a: w_shape, b_shape=(256, 128, 3, 3), (1, 1, 1, 256)
conv3b: w_shape, b_shape=(256, 256, 3, 3), (1, 1, 1, 256)
conv4a: w_shape, b_shape=(512, 256, 3, 3), (1, 1, 1, 512)
conv4b: w_shape, b_shape=(512, 512, 3, 3), (1, 1, 1, 512)
conv5a: w_shape, b_shape=(512, 512, 3, 3), (1, 1, 1, 512)
conv5b: w_shape, b_shape=(512, 512, 3, 3), (1, 1, 1, 512)
fc6: w_shape, b_shape=(1, 1, 4096, 8192), (1, 1, 1, 4096)
fc7: w_shape, b_shape=(1, 1, 4096, 4096), (1, 1, 1, 4096)
fc8: w_shape, b_shape=(1, 1, 101, 4096), (1, 1, 1, 101)

**Comment by chuckcho** [comment]
hi, this was work in progress, and now obsolete and will be closed soon. please refer to https://github.com/axon-research/c3d-keras
